### DISCUSSION TITLE
Comment procéder à la segmentation des double pages

### DISCUSSION THREAD

**Discussion by emchateau** [anrExperts/adire#32] [Q&A]
Lors de la réunion du 21 novembre, nous avons rediscuté de la segmentation des double vues qu’il serait souhaitable d’opérer préalablement au traitement du corpus.

J’avais l’impression que Kraken n’était pas un outil très adapté pour la tâche que nous voulions faire car il est surtout centré sur la détection de zones de texte. D’autres outils de détection de mise en page ou de segmentation semblaient être plus adaptés pour isoler les pages.

Cependant en y repensant, deux approches seraient éventuellement possibles :
- une approche fondée sur la détection des pages
- une approche qui extrapolerait les marges à partir des blocs de texte.

La seconde approche serait peut être implémentable avec Kraken.

Qu'en penses-tu @sardinecan ?

**Comment by rcarvais** [comment]
indépendamment de la difficulté technique, je reste très étonné qu'il n'existe pas une possibilité simple de séparer automatiquement deux pages qui sont organisées quant à leur remplissage de la même manière !

**Comment by emchateau** [comment]
ça existe forcément, c’est pour ça que nous devons chercher.

**Comment by emchateau** [comment]
J'ai fait un premier repérage sur COCO et repéré des solutions logicielles https://github.com/publicarchi/elementum/blob/master/nMLAnnotation.md

**Comment by emchateau** [comment]
Robert a demandé conseil à une collègue qui lui a suggéré AbbyFineReader pour extraire la page. Le résultat est comparable à la solution ScanTaylor proposée par Alix : cela créée une nouvelle image. Or, nous en avons 50 000 et cela nous obligerait à tout recharger sur le site. Il serait donc préférable de travailler avec des coordonnées comme nous l’avons discuté.

Comme je me suis souvenu que nous avons accès à Abby FindeReader par l’intermédiaire de Sharedocs même s’il s’agit d’un outil propriétaire, j'ai fait un test d’extraction au format XML Alto qui nous ramènera peut être les coordonnées de la page. Par contre je ne suis pas trop sûr de comment visualiser ensuite.

**Comment by emchateau** [comment]
Voici le résultat des essais en XMLAlto https://sharedocs.huma-num.fr/wl/?id=KSAlkubD4Q5MAJ19zd4rMo40iPmbwcUf
Peux-tu jeter une œil pour voir si on peut visualiser les blocs de page @sardinecan ? Il faut peut être trouver un outil qui visualiser les zones Alto. Peut-être possible dans eScriptorium.

**Reply by sardinecan** [reply]
J'utilise [altoviz](http://emory-lits-labs.github.io/altoviz/), mais par contre, j'ai pas l'impression que Abby sorte du Alto ?

**Reply by sardinecan** [reply]
il existe cependant des lib. qui permettent de passer de Abby à Alto [abby-to-alto](https://github.com/Mewel/abbyy-to-alto)

**Reply by sardinecan** [reply]
pleins de ressources qui pourraient nous servir ici [ocr-conversion](https://github.com/cneud/ocr-conversion)

**Comment by emchateau** [comment]
En effet, cela n’a pas vraiment l’air d’être du Alto, je m’en suis aperçu en travaillant sur la question avec une étudiante sur un autre projet...
